tests/fake_backends/test_fake_deneb.py

```diff
@@ -20,19 +20,27 @@
 from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister, transpile
 from qiskit_aer.noise.noise_model import NoiseModel
 
+from iqm.iqm_client import CircuitTranspilationError, CircuitValidationError, ExistingMoveHandlingOptions
 from iqm.qiskit_iqm import IQMCircuit, transpile_to_IQM
 from iqm.qiskit_iqm.fake_backends.fake_deneb import IQMFakeDeneb
+from iqm.qiskit_iqm.iqm_backend import IQMTarget
 
 
 def test_iqm_fake_deneb():
     backend = IQMFakeDeneb()
-    assert backend.num_qubits == 7
+    assert backend.num_qubits == 6
     assert backend.name == "IQMFakeDenebBackend"
 
 
-def test_iqm_fake_deneb_connectivity(deneb_coupling_map):
+def test_iqm_fake_deneb_connectivity():
```

Comment:
Why is this no longer refering to the `deneb_coupling_map`?

Reply:
I think you changed it yourself :)